packages/coq-serapi/coq-serapi.8.19.0+0.19.3/opam

```diff
@@ -0,0 +1,55 @@
+opam-version: "2.0"
+maintainer:   "e@x80.org"
+homepage:     "https://github.com/ejgallego/coq-serapi"
+bug-reports:  "https://github.com/ejgallego/coq-serapi/issues"
+dev-repo:     "git+https://github.com/ejgallego/coq-serapi.git"
+license:      "LGPL-2.1-or-later"
```

Comment:
The repo's LICENSE file is complicated. Is it correct that this here package's tarball contains only LGPL.2.1+ code?

Reply:
The tarball contains in addition to SerAPI code, which is LGPL2.1+:
- a modified version of JS sexp printer, which is MIT
- the test cases, which have several licenses, all open source
- the documentation, which is dual-licensed